src/server/api/trpc.ts

```diff
@@ -106,3 +106,7 @@ export const protectedProcedure = t.procedure.use(({ ctx, next }) => {
     },
   });
 });
+
+
+// TYPES
+export type Context = Awaited<ReturnType<typeof createTRPCContext>>;
```

Comment:
nit: Can we get the actual type here instead of using ReturnType?

Reply:
ReturnType simply gives us the type of the function here, which can vary as we modify it.
[Here](https://trpc.io/docs/server/context) is the documentation defining context type the same way